Sources/RouterServiceInterface/FlaggableFeature.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+public protocol FlaggableFeature: Feature {
+    func isEnabled() -> Bool
+    
+    func buildFallback(
```

Comment:
What do you think of this being part of the Feature itself? I think this is useful enough to not need a separate protocol :)

For the fallback, I was thinking of delegating to another feature or route instead of a view controller. I think it could make more sense given that if the feature itself is disabled, it's kinda weird for it to provide another VC.

```swift
public protocol Feature
    // bla bla bla
    func isEnabled() -> Bool
    func fallback(forRoute: Route) -> Feature.Type? // or another Route?
}

public extension Feature {
    func isEnabled() -> Bool { return true }
    func fallback(forRoute: Route) -> Feature.Type? { return nil }
}
```

Then the features that need this behavior can just override these properties 🙂  What do you think?

Reply:
Oh! It really makes a better fit! I got it.